examples/middleware-matcher/middleware.js

```diff
@@ -11,5 +11,8 @@ export default function middleware(req) {
 }
 
 export const config = {
-  matcher: ['/public/disclaimer', '/((?!public|static).*)'],
+  matcher: [
+    '/disclaimer', // match a single, specific page
+    '/((?!public|static).*)' // match all paths not starting with 'public' or 'static'
+  ],
```

Comment:
```suggestion
  matcher: [
    '/disclaimer', // match a single, specific page
    '/((?!public|static).*)', // match all paths not starting with 'public' or 'static'
  ],
```

https://github.com/vercel/next.js/actions/runs/3042937961/jobs/4901675651#step:6:434